jerry-core/jrt/jrt.h

```diff
@@ -227,57 +227,58 @@ inline void *operator new (size_t, void *where)
 } /* operator new */
 
 /**
- * Read data of specified type (T) from specified buffer
+ * Read data of specified type from specified buffer
  *
  * Note:
  *      Offset is in-out and is incremented if the read operation completes successfully.
  *
- * @return true, if read was successful, i.e. offset + sizeof (T) doesn't exceed buffer size,
+ * @return true, if read was successful, i.e. offset + sizeof (data) doesn't exceed buffer size,
  *         false - otherwise.
  */
-template<typename T>
-bool __attr_always_inline___
+
+inline bool
 jrt_read_from_buffer_by_offset (const uint8_t *buffer_p, /**< buffer */
```

Comment:
Perhaps, now it is better to move implementations of the functions from header to a module like `jrt.cpp`.